README.md

```diff
@@ -117,3 +132,49 @@ class Config:
     ignored: str = field(metadata=dict(ignore_arg=True))
 ```
 When indicated, this field is not added to the parser and cannot be overridden with an argument.
+
+### `help`
+
+Provide a brief description of the field, used in the help messages generated by argparse.
+For example, calling `your_program -h` with the dataclass below,
+
+```python
+from dataclasses import dataclass, field
+
+@dataclass
+class Config:
+  an_integer: int = field(metadata=dict(help="any integer you like"))
+```
+
+would result in a message like:
+
+```text
+usage: your_program -h --an-integer AN_INTEGER
```

Comment:
```suggestion
usage: your_program -h
```

why is -h and --an-integer both provided here? does that make sense?

Reply:
nevermind, I see now that this is the resulting output

Reply:
Good one - that actually wasn't fully correct